mm2src/mm2_main/tests/mm2_tests/tendermint_tests.rs

```diff
@@ -825,8 +823,278 @@ mod swap {
         block_on(trade_base_rel_tendermint(
             mm_bob,
             mm_alice,
-            "IRIS-TEST",
-            "RICK",
+            "NUCLEUS-TEST",
+            "DOC",
+            1,
+            2,
+            0.008,
+        ));
+    }
+
+    #[test]
+    fn swap_doc_with_nucleus() {
+        let bob_passphrase = String::from(BOB_PASSPHRASE);
+        let alice_passphrase = String::from(ALICE_PASSPHRASE);
+
+        let coins = json!([nucleus_testnet_conf(), doc_conf()]);
+
+        let mm_bob = MarketMakerIt::start(
+            json!({
+                "gui": "nogui",
+                "netid": 8999,
+                "dht": "on",
+                "myipaddr": env::var("BOB_TRADE_IP") .ok(),
+                "rpcip": env::var("BOB_TRADE_IP") .ok(),
+                "canbind": env::var("BOB_TRADE_PORT") .ok().map (|s| s.parse::<i64>().unwrap()),
+                "passphrase": bob_passphrase,
+                "coins": coins,
+                "rpc_password": "password",
+                "i_am_seed": true,
+            }),
+            "password".into(),
+            None,
+        )
+        .unwrap();
```

Comment:
You can use `Mm2TestConf::seednode` and `MarketMakerIt::start` instead for less boilerplate code.

Reply:
We set various additional fields (myipaddr, rpcip, canbind, i_am_seed, skip_startup_checks) that cannot be configured using `seednode`/`light_node` functions.